.prettierrc

```diff
@@ -1,7 +1,7 @@
 {
     "useTabs": false,
-    "tabWidth": 2,
```

Comment:
any particular reason for changing the tabWidth in the entire project?

Reply:
@krishnakalluri and I are going to work on a tslint/eslint config and publish it to Twilio repo; Twilio mostly uses 4 spaces for spacing.